core/server/services/mega/mega.js

```diff
@@ -108,7 +108,7 @@ const serialize = async (model) => {
 function createUnsubscribeUrl(member) {
     const siteUrl = urlUtils.getSiteUrl();
     const unsubscribeUrl = new URL(siteUrl);
-    unsubscribeUrl.searchParams.set('action', 'unsubscribe');
+    unsubscribeUrl.pathname = `${unsubscribeUrl.pathname}/unsubscribe`.replace('//', '/');
     unsubscribeUrl.searchParams.set('unsubscribe', member.uuid);
```

Comment:
Not sure we really need a separate param here? Maybe `/unsubscribe/:uuid/` is better?

Reply:
Yeah I agree - seeing as this won't be handled like other "actions" we can probably drop it

Reply:
Thinking about this further I'll leave it as is but I'll change the param to be `?uuid=`. We discussed the ability to unsubscribe from more specific things in the future so params on the base /unsubscribe/ route leaves us more flexibility.